js/src/data/test/reducer.test.js

```diff
@@ -418,87 +413,42 @@ describe( 'reducer', () => {
 	} );
 
 	describe( 'Policy Check', () => {
-		it( 'should return with allowed countries', () => {
+		it( 'should return with policy check info', () => {
 			const data = {
-				policy_check: {
-					allowed_countries: {
-						CA: { currency: 'CAD', name: 'Canada' },
-						US: { currency: 'USD', name: 'United States' },
-					},
+				allowed_countries: {
+					CA: { currency: 'CAD', name: 'Canada' },
+					US: { currency: 'USD', name: 'United States' },
 				},
-			};
-			const action = {
-				type: TYPES.RECEIVE_ALLOWED_COUNTRIES,
-				data,
-			};
-			const state = reducer( prepareState(), action );
-
-			state.assertConsistentRef();
-			expect( state ).toHaveProperty(
-				'mc.policy_check.allowed_countries',
-				data.policy_check.allowed_countries
-			);
-		} );
-
-		it( 'should return with store_ssl', () => {
-			const data = {
-				policy_check: {
-					store_ssl: true,
+				store_ssl: true,
+				payment_gateways: {
+					id: 'wc_custom_pg',
+					title: 'Custom Payment Gateway',
+					method_description: 'Description of the payment gateway',
 				},
+				refund_returns: 'Refund and Returns Policy',
```

Comment:
These values still pass in the test, but they don't match up with what it was changed to in the PHP code. Each of these properties should return a boolean value. Same for the `allowed_countries`.

Reply:
Updated.